### DISCUSSION TITLE
把数据包封装成qq的格式是不是可以突破封锁？

### DISCUSSION THREAD

**Discussion by oaklk** [shadowsocks/shadowsocks-windows#3100] [Ideas]
能否用qq的格式，或者中国任意流行社交软件的封装格式来封装
qq也有点对点方式，这种情形应该不经过中央服务器，
我是外行，只是一个建议，不知道是否可行

**Comment by database64128** [comment]
We welcome new ideas, but I’m afraid this is very much out of scope for issues here. Converting to discussion so anyone can post their ideas and opinions.
